src/components/calcite-shell-panel/calcite-shell-panel.scss

```diff
@@ -1,24 +1,45 @@
-$shell-panel-width: 20vw !default;
-$shell-panel-min-width: 240px !default;
-$shell-panel-max-width: 360px !default;
-
 :host {
-  height: 100%;
   position: relative;
   display: flex;
-  background-color: var(--calcite-app-background-content);
+  align-items: flex-start;
+  background-color: var(--calcite-app-background-transparent);
+  pointer-events: none;
+
+  --calcite-app-shell-panel-width: 20vw;
+  --calcite-app-shell-panel-min-width: 240px;
+  --calcite-app-shell-panel-max-width: 360px;
+  --calcite-app-shell-panel-max-height-small: 40vh;
+  --calcite-app-shell-panel-max-height-medium: 60vh;
+  --calcite-app-shell-panel-max-height-large: 85vh;
 }
 
 .content {
-  width: $shell-panel-width;
-  min-width: $shell-panel-min-width;
-  max-width: $shell-panel-max-width;
+  background-color: var(--calcite-app-background-content);
+  width: var(--calcite-app-shell-panel-width);
+  min-width: var(--calcite-app-shell-panel-min-width);
+  max-width: var(--calcite-app-shell-panel-max-width);
   border-left: 1px solid var(--calcite-app-border);
   border-right: 1px solid var(--calcite-app-border);
   padding: 0;
-  margin: 0;
-  height: 100%;
   overflow: auto;
+  pointer-events: auto;
+}
+
+:host([float-content]) .content {
+  border: 1px solid var(--calcite-app-border);
+  border-radius: var(--calcite-app-border-radius);
+  box-shadow: var(--calcite-app-shadow-2);
+  margin: var(--calcite-app-cap-spacing-half) var(--calcite-app-side-spacing-half);
+  max-height: var(--calcite-app-shell-panel-max-height-medium);
+  height: auto;
+}
+
+:host([float-content][float-content-scale="s"]) .content {
```

Comment:
@asangma can you test this in Edge? I'm not sure if the polyfill will handle it.

Reply:
If it doesn't, might be better to have a `.content--modifier` class.

Reply:
Sounds good.

Reply:
@driskull Done.